Backend/Settings/CalendarSettings.cs

```diff
@@ -0,0 +1,14 @@
+namespace FRITeam.Swapify.Backend.Settings
+{
```

Comment:
Is there some date type for configuration? Two parameters instead of 8

Reply:
there is